debug/analyzer/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2015-2019 Clearmatics Technologies Ltd
```

Comment:
Extraneous line.

I didn't include the copyright headers to the `__init__.py` files since there are basically empty files. I'd add `#!/usr/bin/env python3` though

Reply:
I've generally been putting the she-bang into files that might be run as shell commands, esecially if they don't have the `.py` extension, since it's also a hint to editors.

Are you suggesting we enforce it in all python files?  If so, I'll add a little script to enforce consistency.

Reply:
> I've generally been putting the she-bang into files that might be run as shell commands

Right, let's keep it that way. Adding the hashbang to all (not necessarily executable) files in contrary to the intended use.